From e902f8ad85a3a36b266f5e96bf3dc1ddad9fa18e Mon Sep 17 00:00:00 2001
From: Nicholas Bennett <nicholas.bennett@qt.io>
Date: Wed, 17 Sep 2025 13:12:13 +0300
Subject: [PATCH 475/553] Docs: Fix QAndroidNativeInterface doc bug

Replaced QJniObject with  QtJniTypes::Context

Fixes: QTBUG-140208
Pick-to: 6.8
Change-Id: Id9567ae4b26a457bf8dc614fb29b64ed27b4063e
Reviewed-by: Assam Boudjelthia <assam.boudjelthia@qt.io>
Reviewed-by: Rami Potinkara <rami.potinkara@qt.io>
Reviewed-by: Andrey Filipenkov <decapitator@ukr.net>
(cherry picked from commit db76ba47ffb754ab4d16732347b96c0e0f3e8a70)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/platform/android/qandroidnativeinterface.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/corelib/platform/android/qandroidnativeinterface.cpp b/src/corelib/platform/android/qandroidnativeinterface.cpp
index 680725dc6a1..5771d28f3a9 100644
--- a/src/corelib/platform/android/qandroidnativeinterface.cpp
+++ b/src/corelib/platform/android/qandroidnativeinterface.cpp
@@ -48,8 +48,9 @@ QT_DEFINE_NATIVE_INTERFACE(QAndroidApplication);
 /*!
     \fn QJniObject QNativeInterface::QAndroidApplication::context()
 
-    Returns the Android context as a \c QJniObject. The context is an \c Activity
-    if the most recently started activity object is valid. Otherwise, the context is a \c Service.
+    Returns the Android context as a \c QtJniTypes::Context. The context
+    is an \c Activity if the most recently started activity object is valid.
+    Otherwise, the context is a \c Service.
 
     \since 6.2
 */
-- 
2.51.2

